src/model_editor/ModelEditor.i

```diff
@@ -53,6 +52,13 @@
   }
 %}
 
+#if defined(SWIGCSHARP) || defined(SWIGJAVA)
+%module(directors="1")
```

Comment:
Do you need to turn directors on before the C++ include?  Seems you could leave the C++ include where it was above?